pjsip/include/pjsua2/media.hpp

```diff
@@ -1938,7 +1938,10 @@ struct VideoPreviewOpParam {
     unsigned                windowFlags;
 
     /**
-     * Media format video. If left unitialized, this parameter will not be used.
+     * Media format video. If left uninitialized, this parameter will not be used and 
+     * the capture device will be opened using PJMEDIA wrapper default format, 
+     * e.g: on Android it is PJMEDIA_FORMAT_I420, on iOS it is PJMEDIA_FORMAT_BGRA.
```

Comment:
Actually I was thinking about the default resolution and frame rate info here rather than pixel format, as an initial hint about the video quality :)